src/styles/untiles/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$white-color:#fff;
+$arial-color:#00acdc;
+$bold-weight:700;
+$bludark-color:#060b35;
```

Comment:
There is a typo in the variable name 'bludark-color'. Consider renaming it to 'blue-dark-color'.